deriveapprodi.csl

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="it-IT">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>DeriveApprodi (Italiano)</title>
+    <id>http://www.zotero.org/styles/deriveapprodi</id>
+    <link href="http://www.zotero.org/styles/deriveapprodi" rel="self"/>
+    <link href="http://www.zotero.org/styles/geografie-sbornik-cgs" rel="template"/>
+    <link href="https://forums.zotero.org/discussion/97879/style-request-deriveapprodi" rel="documentation"/>
```

Comment:
There is no publicly available URL for the citation guidelines?

Reply:
Hi, this style is based on the citation guidelines I got from the publisher, that are sent to the prospect authors. Do they have to be publicly available?